lhotse/recipes/iwslt22_ta.py

```diff
@@ -0,0 +1,370 @@
+# Copyright    2023  Johns Hopkins        (authors: Amir Hussein, Matthew Wiesner)

```

Comment:
We usually don't have license in the recipes.

Reply:
No worries, just wanted to give credits to Matthew. Can I keep the authors?

Reply:
I think it's fine to keep it.

Reply:
Done